app/assets/javascripts/modules/collapsible.js

```diff
@@ -5,26 +5,35 @@
   function Collapsible(section){
     this.$section = section;
     this.$clickTarget = this.$section.find('.js-subsection-title');
-    this.$clickTarget.on('click', this.toggle.bind(this));
+    this.$clickTarget.on('click', this.handleClick.bind(this));
     this.addToggle();
 
     this.$section.on('focus', '.js-subsection-body a', this.showSectionWhenLinkFocused.bind(this));
   }
 
+  Collapsible.prototype.handleClick = function handleClick(){
+    this.toggle()
+    this.toggleAttribute()
+  };
+
+  Collapsible.prototype.toggleAttribute = function() {
+    var $button = $(this.$clickTarget[0].firstChild)
```

Comment:
If we pass the `event` from `handleClick`, we can avoid this slightly brittle looking code:

```js
this.toggleAttribute(e.target)

// ...

Collapsible.prototype.toggleAttribute = function(button) {
```

Reply:
it's worth remembering users can also click the toggle span, in which case the target is different but we still need to only toggle the button's attribute

Reply:
it's worth remembering users can also click the toggle span, in which case the target is different but we still need to only toggle the button's attribute

Reply:
replaced with a less brittle querySelector has the parent H2 is already available as the $clickTarget